aspnetcore/tutorials/first-mvc-app/adding-model.md

```diff
@@ -3,20 +3,18 @@ title: Part 4, add a model to an ASP.NET Core MVC app
 author: rick-anderson
 description: Part 4 of tutorial series on ASP.NET Core MVC.
 ms.author: riande
-ms.date: 11/16/2020
+ms.date: 01/30/2021
 no-loc: [appsettings.json, "ASP.NET Core Identity", cookie, Cookie, Blazor, "Blazor Server", "Blazor WebAssembly", "Identity", "Let's Encrypt", Razor, SignalR]
 uid: tutorials/first-mvc-app/adding-model
 ---
 
 # Part 4, add a model to an ASP.NET Core MVC app
 
-By [Rick Anderson](https://twitter.com/RickAndMSFT) and [Tom Dykstra](https://github.com/tdykstra)
+In this section, classes are added for managing movies in a database. These classes are the "**M**odel" part of the **M**VC app.
 
-In this section, you add classes for managing movies in a database. These classes will be the "**M**odel" part of the **M**VC app.
+These model classes are used with [Entity Framework Core](/ef/core) (EF Core) to work with a database. EF Core is an object-relational mapping (ORM) framework that simplifies the data access code that you have to write.
 
-You use these classes with [Entity Framework Core](/ef/core) (EF Core) to work with a database. EF Core is an object-relational mapping (ORM) framework that simplifies the data access code that you have to write.
-
-The model classes you create are known as POCO classes (from **P**lain **O**ld **C**LR **O**bjects) because they don't have any dependency on EF Core. They just define the properties of the data that will be stored in the database.
+The model classes you create are known as POCO classes (from **P**lain **O**ld **C**LR **O**bjects). POCO classes don't have any dependency on EF Core. They only define the properties of the data to be stored in the database.
```

Comment:
I thought about this one and purposely left this "you" in.  We have scaffolding that creates classes.  It could be clearer who is creating the classes.